README.md

```diff
@@ -15,9 +15,15 @@ OpenPBR Surface is an open standard hosted by the [Academy Software Foundation](
 
 * **[White paper](https://academysoftwarefoundation.github.io/OpenPBR/)**
 * **[Parameter reference](https://academysoftwarefoundation.github.io/OpenPBR/#parameterreference)**
-* **[Reference implementation](reference/open_pbr_surface.mtlx)** – written in [MaterialX](https://materialx.org/)
```

Comment:
In some ways, I still prefer the original wording, which makes it clear that there's just a single reference implementation of OpenPBR.  Maybe we can consider this proposed change further in the future, and focus on the Resources section for this change?

Reply:
I don't think there really is (or should be) an official "reference implementation", as the spec itself is supposed to define how to implement the model. The MaterialX implementation is also not really currently a good reference in my opinion, as it's both hard to understand and also uses a bunch of approximations (for example, my implementation is the viewer is actually more faithful to the spec).

Even so, it's the only official reference implementation we have, so we should flag it as such in the main "Specification" section. I don't see a problem with referring to it as "MaterialX reference implemention", not *the* reference implementation.

Reply:
@portsmouth This sounds like a deeper discussion that you and I should have, and I'd recommend that we separate this from the current set of proposed updates to our front page.  :)

Reply:
OK, so the change you want is just that we put back
>* **[Reference implementation](reference/open_pbr_surface.mtlx)** – written in [MaterialX](https://materialx.org/)

Correct?  If so, I think the difference doesn't matter much, so fine. (It's "a" reference implementation, not "the" one, from my point of view, but we don't say either so no problem).

Reply:
Yes, I'd just recommend leaving this text alone for now, since it's served us well up to this point.  If you're interested in reframing our approach to reference implementations in OpenPBR, then we should discuss that directly (e.g. in TSC meetings or on Slack), and we can make adjustments to our framing based on the findings of that discussion.

Reply:
I don't think i'm proposing a reframe.. I was just stating it the way it is - we don't say in the spec that the MaterialX implementation is the ground truth. Maybe there's some difference of opinion about what a "reference implementation" means then. I'm OK with the leaving the text as is though.